src/env.c

```diff
@@ -93,13 +89,7 @@ static struct {
     // Exit callbacks
     sel4runtime_exit_cb *exit_cb;
     sel4runtime_pre_exit_cb *pre_exit_cb;
-} env = {
-    /*
-     * Initialise the initial thread as referring to the global thread
-     * object.
-     */
-    .initial_thread_tls_base = (uintptr_t)NULL,
-};
+} env; /* In BSS: All fields initialised to zero */
```

Comment:
It's not clear to me why this comment was added here.

Reply:
I got rid of the unnecessary initialisation to NULL: as `env` no longer has any initilialisations, itll be allocated in the BSS and be all zeros